docs/api/openapi.yaml

```
@@ -1538,6 +1670,179 @@ paths:
               required:
               - linking_code
         required: true
+  "/api/v1/auth/sso_link":
```

YAMLlint (1.38.0)

[error] duplication of key "/api/v1/auth/sso_link" in mapping

(key-duplicates)